cmd/collector/app/span_processor.go

```diff
@@ -164,7 +164,7 @@ func (sp *spanProcessor) ProcessSpans(mSpans []*model.Span, options processor.Sp
 	for i, mSpan := range mSpans {
 		ok := sp.enqueueSpan(mSpan, options.SpanFormat, options.InboundTransport)
 		if !ok && sp.reportBusy {
-			return nil, tchannel.ErrServerBusy
+			return nil, fmt.Errorf("server busy")
```

Comment:
nit: this should be a dedicated error (similar to TraceNotFound in the query service) and the gRPC handler should translate it it to gRPC-specific status code. But it's not critical as there's no special processing in the agent for this.

Reply:
Actually, might be still critical for us because "busy" is understood by the service mesh

Reply:
PR https://github.com/jaegertracing/jaeger/pull/2314